README.md

```diff
@@ -157,8 +163,8 @@ $proxy->demo()->then(
 
 ## Install
 
-The recommended way to install this library is [through composer](packagist://getcomposer.org).
-[New to composer?](packagist://getcomposer.org/doc/00-intro.md)
+The recommended way to install this library is [through composer](http://getcomposer.org).
+[New to composer?](http://getcomposer.org/doc/00-intro.md)
```

Comment:
Looks like your feature branch is based on an old version of the master branch. Can you rebase and push your changes?